### PR TITLE
Add LoanDolphin

### PR DESCRIPTION
**Pull request by donbonifacio**
Add LoandDolphin to fintech category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [LoanDolphin](https://loandolphin.com.au/) | Home loan marketplace. | `Lisboa` `Remote` |
```
Comment (marmelo) [inline README.md]: Please switch order with Feedzai.  
Companies must be sorted alphabetically.

Thanks!

Comment (donbonifacio) [reply README.md]: Missed having PRs reviewed by you buddy. :)

Comment (donbonifacio) [reply README.md]: @marmelo done 👍

**Comment by marmelo** [comment]
Thanks @donbonifacio! :rocket:

**Pull request merged**